### PR TITLE
My appointments

### PR DESCRIPTION
**Pull request by braincee**
### _**In this pull request:**_

- I created a component to render all the appointments of a user
- I created a functionality to delete an appointment

### CODE REVIEW

File: src/components/Appointments/MyAppointments.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const myAppointments = selfAppoitments();
+
+  return (
+    <section>
+      <h1>{User.name}</h1>
+      {
+      myAppointments.length === 0 ? (
+        <div>
+          <h2>No Appointments Available</h2>
+        </div>
```
Comment (Bria222) [inline src/components/Appointments/MyAppointments.js:41]: nice idea returning  h2 response  but this could be better if you returned it conditionary

**Review by Bria222 — APPROVED** [review]
# Status: Approved 🟢🟢🟢♻️♻️♻️

Hi @braincee ,

Good job👍

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly
✅ All linter checks passed
✅ Nice api call 
well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by braincee** [comment]
Thanks very much, @Bria222 

thank-you

**Review by Meltrust — COMMENTED** [review]
_note: There was no PR or link to the backend project.  This PR was already merged before review.  For your next review request, kindly open a PR on your backend project with all the content, it should point from dev to main.  Please, include the link to the new PR in this project so the next reviewer can make a proper review._ 

Hi @braincee,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!










#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Comment by Meltrust** [comment]
Hi @braincee

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that contains a link to a PR instead of an issue with a Kanban board.  Please, include the correct link.  If this was a project review, please request from the correct project button.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**